examples/maildev/index.js

```diff
@@ -12,11 +12,47 @@ module.exports = async function (fastify, options) {
   fastify.register(fastifyMail, { pov: { engine: { nunjucks } }, transporter })
 
   fastify.get('/sendmail', async (req, reply) => {
-    const recipients = ['test@example.com']
-    const templates = './templates'
-    const context = { name: 'Test Name', sender: 'sender@example.com' }
+    const { name } = req.query
+    const templatePath = './templates'
```

Comment:
move down below the first parameter configuration (these bits are secondary and optional, so need to be demoted over the required/non-optional param)

Reply:
done